packages/fluent/scss/coloreditor/_layout.scss

```diff
@@ -112,24 +112,6 @@
             .k-colorpalette {
                 align-self: center;
             }
-
-            .k-colorpalette-tile {
-                width: var( --kendo-color-preview-width, $kendo-color-palette-tile-width );
```

Comment:
--kendo-color-preview css var does not seem to be relevant for the size of the tile. I think we should replace that.